### PR TITLE
Update the Spanish language

### PR DESCRIPTION
**Pull request by CarlosBarrazaE**
Actualice el idioma español
Agrege una plantilla
Corregí un pequeño error.

### CODE REVIEW

**Comment by SmallJoker** [comment]
I don't speak Spanish, but the changes look good overall. Will merge within the next few days unless there are any objections.

File: locale/unified_inventory.es.tr [hunk 1]
```diff
@@ -1,38 +1,100 @@
...
+invisible=invisible
+visible=visible
+@1 display of waypoint coordinates=Visualizar coordenadas del punto @1
+Disable=Deshabilitado
+Enable=Habilitado
+Change color of waypoint display=Cambiar el color del punto
+Edit waypoint name=Editar nombre del punto
+Waypoint active=Punto activo
+Waypoint inactive=Punto inactivo
+Finish editing=Edición Finalizada
```
Comment (Panquesito7) [inline locale/unified_inventory.es.tr]:
```suggestion
Finish editing=Terminar edición
```

File: locale/unified_inventory.es.tr [hunk 2]
```diff
@@ -1,38 +1,100 @@
...
+
+First page=Primera página
+Back three pages=Volver tres páginas
+Back one page=Volver una página
+Forward one page=Avanzar una página
+Forward three pages=Avanzar tres páginas
+Last page=Ultima Pagina
+Search=Buscar
+Reset search and display everything=Limpiar la busqueda y mostrar todo
+No matching items=No se encontro ningun elemeto
```
Comment (Panquesito7) [inline locale/unified_inventory.es.tr]:
```suggestion
No matching items=No se encontraron elementos
```

File: locale/unified_inventory.es.tr [hunk 3]
```diff
@@ -1,38 +1,100 @@
...
+Reset search and display everything=Limpiar la busqueda y mostrar todo
+No matching items=No se encontraron elementos
+No matches.=No hay resultados.
+Page=Página
+@1 of @2=@1 de @2
+Filter=Filtro
+
+# bags.lua
+
+Bags=Bolsos
```
[the remark this answers is not in the document]
Comment (CarlosBarrazaE) [reply locale/unified_inventory.es.tr:96]: For the length of the word, it does not reach the button.

File: locale/unified_inventory.es.tr [hunk 4]
```diff
@@ -1,38 +1,100 @@
...
+
 Can use the creative inventory=Puede usar el inventario creativo
+Forces Unified Inventory to be displayed in Full mode if Lite mode is configured globally=Obliga al Inventario Unificado a mostrarse en modo Completo si el modo Simple está configurado globalmente
+Crafting Grid=Cuadricula de Elaboración
 Crafting Guide=Guía de Elaboración
-Set home position=Posición en el mundo
-Home position set to: @1=Posición de hogar cambiada a: @1
-You don't have the "home" privilege!=¡No tienes el privilegio "home"!
-Time of day set to 6am=Hora del día cambiada a 6AM
+Set home position=Establecer posición del hogar
```
[the remark this answers is not in the document]
Comment (CarlosBarrazaE) [reply locale/unified_inventory.es.tr:39]: The starting position is not the same as the position of the home.
"Posición de inicio" is Spawn

File: locale/unified_inventory.es.tr [hunk 5]
```diff
@@ -1,38 +1,100 @@
...
 Result=Resultado
-To craft grid:=Copiar al cuadro de elaboración
+Ingredient=Ingrediente
+Show next recipe=Mostrar la siguiente receta
+Show next usage=Mostrar el siguiente uso
+Show previous recipe=Mostrar la receta anterior
+Show previous usage=Mostrar el uso anterior
+@1 (@2)=@1 (@2)
+Give me:=Dame:
+This recipe is too@nlarge to be displayed.=Esta receta es demaciado@ngrande para ser mostrada.
```
[the remark this answers is not in the document]
Comment (CarlosBarrazaE) [reply locale/unified_inventory.es.tr]: Corrected

File: locale/unified_inventory.es.tr (hunk 4 above)
[the remark this answers is not in the document]
Comment (raistlin) [reply locale/unified_inventory.es.tr:39]: Well, really when you die, you will be spawned on your home so ...

If you insist, then please change this to  "Establecer posición de la casa"and te next one to "Posición de la casa cambiada a: @1" to be consistent with "Ir a casa", its messy to have "hogar" sometimes and "casa" in others ... and also "hogar" can be  misunderstood with "fuego" :-)

[the remark this answers is not in the document]
Comment (CarlosBarrazaE) [reply locale/unified_inventory.es.tr]: Done,  I had no idea that "hogar" also meant "fuego", maybe in other countries it is more common.

**Pull request merged**